### PR TITLE
ci: add /test GitHub comment as a build trigger

### PR DESCRIPTION
**Pull request by kuisathaverat**
it adds the GitHub comment `/test` as a build trigger 

related to https://github.com/elastic/package-storage/issues/151

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
+    issueCommentTrigger('(?i)().*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*|^/test$)')
```
Comment (kuisathaverat) [inline .ci/Jenkinsfile:27]:
```suggestion
    issueCommentTrigger('(?i)(.*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*|^\\/test$)')
```

**Comment by jonathan-buttner** [comment]
This is great! Any chance we could throw it in a readme somewhere too? I'm bound to forget 😄

**Pull request merged**